get_gerrit_last_comment_by_data.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+if ! command -v jq >/dev/null 2>&1; then
+    echo >&2 "I require jq but it's not installed. Aborting."
+    exit 1
+fi
+
+usage() { 
+    echo "Usage: $0 [-h|--gerrit_host HOST] [-p|--port NUM] [-c|--change_number NUM] [-s|--ssh_key_path PATH] [-u|--gerrit_username USERNAME]"
+    exit 2
+}
+
+declare gerrit_host port change_number ssh_key_path gerrit_username
+
+while [ $# -gt 0 ]; do
+    case $1 in
+        -h|--gerrit_host)
+            gerrit_host=$2
+            shift
+            ;;
+        -p|--port) 
+            port=$2
+            shift
+            ;;
+        -c|--change_number) 
+            change_number=$2
+            shift
+            ;;
+        -s|--ssh_key_path)
+            ssh_key_path=$2
+            shift
+            ;;
+        -u|--gerrit_username)
+            gerrit_username=$2
+            shift
+            ;;
+        --)
+            shift
+            break
+            ;;
+        *)
+            usage
+    esac
+    shift
+done
+
+if [[ -z $port ]]; then
+    port=22
+fi
+
+if [ -z "$gerrit_host" ]; then
+    echo 'Missing -h parameter: --gerrit_host'
+    exit 2
+fi
+
+if [ -z "$change_number" ]; then
+    echo 'Missing -c parameter: --change_number'
+    exit 2
+fi
+
+if [ -z "$ssh_key_path" ]; then
```

Comment:
That should not be mandatory. In many systems ~/.ssh/config is sufficient enough to connect to the host without need to specify ssh key on the command line. So default what's configured and if it's specified here then use it.

Reply:
Secondly you are not using this parameter anywhere in this script.

Reply:
Change done.

Now the `ssh_key_path` is optional and is used if we pass it.